File: main-service/src/main/kotlin/sk/stuba/fei/api/msus/dp/mainservice/model/payload/request/VerificationRequest.kt
package sk.stuba.fei.api.msus.dp.mainservice.model.payload.request

import sk.stuba.fei.api.msus.dp.mainservice.model.VerificationParameters
import sk.stuba.fei.api.msus.dp.mainservice.model.payload.request.dto.ModalityVerificationRequestDto

data class VerificationRequest(
    val criminalId: String,
    val modality: ModalityVerificationRequestDto,
    val verificationParameters: VerificationParameters?
)

File: dp-client/src/app/model/payload/request/verification-request.ts
import {VerificationParameters} from '../../verification-parameters';
import {ModalityVerificationRequestDto} from './dto/modality-verification-request-dto';

export interface VerificationRequest {
  criminalId: string;
  modality: ModalityVerificationRequestDto;
  verificationParameters: VerificationParameters | null;
}
